packages/mattermost-redux/src/client/client4.ts

```diff
@@ -1926,9 +1926,9 @@ export default class Client4 {
         );
     };
 
-    getPostThread = (postId: string, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false) => {
+    getPostThread = (postId: string, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false, fromCreateAt?: number, fromPost?: string, direction?: 'up'|'down', perPage?: number) => {
```

Comment:
Not something to change now, but we should consider using an object to pass large numbers of optional arguments through these, especially since the Client4 methods are public so we have to maintain compatibility with these

Reply:
Yep I was thinking about that as well.
As I see it the signature could be `getPostThread(postId: string, options: PostThreadOptions)`